src/main/java/frc/team5115/Robot/RobotContainer.java

```diff
@@ -178,6 +183,7 @@ public void teleopPeriodic() {
         // manual climber
         if(climber.isDeployed()) climber.setBoth(joyManips.getRawAxis(1));
 
+        a.periodic1();
```

Comment:
Do you need the results of this function? Is the array of errors useful?

Reply:
I think we just used that to see if we could run the aim and range during teleop; I'm pretty sure it's commented out now